woocommerce-admin.php

```diff
@@ -214,6 +214,7 @@ function wc_admin_plugins_loaded() {
 	require_once WC_ADMIN_ABSPATH . 'includes/notes/class-wc-admin-notes-historical-data.php';
 	require_once WC_ADMIN_ABSPATH . 'includes/notes/class-wc-admin-notes-order-milestones.php';
 	require_once WC_ADMIN_ABSPATH . 'includes/notes/class-wc-admin-notes-mobile-app.php';
+	require_once WC_ADMIN_ABSPATH . '/includes/notes/class-wc-admin-notes-welcome-message.php';
```

Comment:
Small change here: can you remove the leading `/`? :slightly_smiling_face: 
```suggestion
	require_once WC_ADMIN_ABSPATH . 'includes/notes/class-wc-admin-notes-welcome-message.php';
```

Reply:
Sorry, My mistake implemented the change.